### PR TITLE
rook: fix cli env var

### PR DESCRIPTION
**Pull request by leseb**
**Description of your changes:**

Previously, the variables were always initialized to True, thus their
value was never modified.
By simply making a bool it will be modified.

Signed-off-by: Sébastien Han <seb@redhat.com>

**Checklist:**

- [ ] **CommitLint Bot**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

[test ceph]

### CODE REVIEW

File: pkg/operator/ceph/operator.go [hunk 1]
```diff
@@ -56,10 +56,11 @@ var provisionerConfigs = map[string]string{
 }
 
 var (
-	// Whether to enable the flex driver. If true, the rook-ceph-agent daemonset will be started.
-	EnableFlexDriver = true
-	// Whether to enable the daemon for device discovery. If true, the rook-ceph-discover daemonset will be started.
-	EnableDiscoveryDaemon = true
+	// EnableFlexDriver Whether to enable the flex driver. If true, the rook-ceph-agent daemonset will be started.
+	EnableFlexDriver bool
```
Comment (travisn) [inline pkg/operator/ceph/operator.go:60]: This field is set to `false` by translating the operator env var to the command line [here](https://github.com/rook/rook/blob/master/cmd/rook/ceph/operator.go#L46). I wasn't able to repro the issue of the flex agent always starting. Shall we close this issue? Or how did you repro the isuse?

**Comment by travisn** [comment]
Looks like this is not needed.

**Comment by leseb** [comment]
> Looks like this is not needed.


Did you test again?

**Comment by travisn** [comment]
@leseb Yes, I haven't seen any issues with the flex driver being disabled by default

**Comment by leseb** [comment]
> @leseb Yes, I haven't seen any issues with the flex driver being disabled by default

Ok I don't repro anymore either, strange, anyway!

**Pull request closed without merging**